storage/content/fio-pod-pv.json

```diff
@@ -137,7 +137,7 @@
 	    "name": "VOLUME_CAPACITY",
 	    "displayName": "Volume Capacity",
 	    "description": "Volume space available for data, e.g. 512Mi, 2Gi",
-	    "value": "10Gi",
+	    "value": "1000Gi",
```

Comment:
Instead of changing the default value, I would do it via the shell task of oc-process command since it is already a parameter.

Reply:
@hongkailiu yeah agree that is better way will make changes accordingly